gym/spec/xcodebuild_fixes/generic_archive_fix_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'gym/xcodebuild_fixes/generic_archive_fix'
+
+describe Gym do
+  describe Gym::XcodebuildFixes do
+    let (:watch_ipa) { 'spec/fixtures/xcodebuild_fixes/ios watch_app.app/Info.plist' }
```

Comment:
It might be nice to convert the plist file to text version of plist files so that we can view them easily on github